### PR TITLE
basic homepage

### PR DESCRIPTION
**Pull request by acjc**
@beatrizz

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -3,124 +3,114 @@ import 'package:english_words/english_words.dart';
...
+      )
     );
   }
 }
 
-class RandomWords extends StatefulWidget {
+class HomePage extends StatefulWidget {
   @override
-  createState() => new RandomWordsState();
+  State<StatefulWidget> createState() {
```
Comment (beatrizz) [inline lib/main.dart]: Doesn't the compact syntax work?

File: lib/main.dart [hunk 2]
```diff
@@ -3,124 +3,114 @@ import 'package:english_words/english_words.dart';
...
+    void createRoom() {
+      print('CREATE ROOM');
+    }
 
-    return new ListTile(
-      title: new Text(
-        pair.asPascalCase,
-        style: _biggerFont,
+    Widget createRoomButton = new Container(
+      padding: const EdgeInsets.all(32.0),
```
Comment (beatrizz) [inline lib/main.dart]: There may be a way of styling things in one go, e.g. set the same paddings everywhere, but we can work it out later :)

File: lib/main.dart [hunk 3]
```diff
@@ -3,124 +3,114 @@ import 'package:english_words/english_words.dart';
...
+      child: new RaisedButton(
+        child: const Text(
+          'CREATE ROOM',
+          style: const TextStyle(color: Colors.white, fontSize: 16.0)
+        ),
+        onPressed: createRoom,
+        color: Theme.of(context).primaryColor,
```
Comment (beatrizz) [inline lib/main.dart]: Unfortunately white on deep orange isn't accessible enough: https://webaim.org/resources/contrastchecker/
I've taken the deep orange value from here (the one with nothing between brackets, #FF5722): https://docs.flutter.io/flutter/material/Colors/deepOrange-constant.html
We can create an issue and pick a colour later.

File: lib/main.dart [hunk 4]
```diff
@@ -77,13 +71,17 @@ class HomePageState extends State<HomePage> {
         new Row(
           mainAxisAlignment: MainAxisAlignment.spaceEvenly,
           children: [
-            buildArrowColumn(Icons.arrow_back, () => setState(() {
-              if (_numPlayers > minPlayers) _numPlayers--;
-            })),
+            buildArrowColumn(
+                Icons.arrow_back,
+                () => setState(() {
+                      if (_numPlayers > minPlayers) _numPlayers--;
```
Comment (beatrizz) [inline lib/main.dart]: ifs with no brackets make me sad buuuuut ok, this is unlikely to ever do anything else

File: lib/main.dart [hunk 5]
```diff
@@ -3,124 +3,114 @@ import 'package:english_words/english_words.dart';
...
+            )
+          ],
+      );
+    }
+
+    Widget numPlayersText = new Container(
+      padding: const EdgeInsets.all(32.0),
+      child: new Text(
+        _numPlayers.toString(),
+        style: new TextStyle(
```
Comment (beatrizz) [inline lib/main.dart]: The other TextStyles have the value in the same line and they're const (lines 70 and 101). This one should be consistent, unless there's a reason for it to be different that I'm missing.

File: lib/main.dart [hunk 6]
```diff
@@ -30,7 +30,7 @@ class HomePage extends StatefulWidget {
 }
 
 class HomePageState extends State<HomePage> {
-  var _numPlayers = minPlayers;
+  int _numPlayers = minPlayers;
```
Comment (beatrizz) [inline lib/main.dart]: Can we make the consts at the top ints as well?

File: lib/main.dart (hunk 2 above)
Comment (acjc) [reply lib/main.dart]: Yes, I hope so, but I haven't got up to the bit in the docs where it says yet!

File: lib/main.dart (hunk 3 above)
Comment (acjc) [reply lib/main.dart]: I expect we won't actually have an app bar; games don't usually.

File: lib/main.dart (hunk 5 above)
Comment (acjc) [reply lib/main.dart]: Yes, well spotted, it can be const, it's just the Text object above which obviously can't.

**Pull request merged**